coalib/results/Diff.py

```diff
@@ -244,10 +245,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._original
 
-    @property
-    def modified(self):
+    def _raw_modified(self):
```

Comment:
I thought keeping `_file` would be a good idea if a un-linebreak-ified version of the file is required somewhere. However, it seems it is not called anywhere outside of Diff, so we could ditch it.

Not sure I understand the other suggestion (or rather, the advantage of it, assuming we keep EOF-state). 
Initially adding the linebreaks will complicate the modification functions, because we have to check for addition/modification/delection locations to preserve the EOF-state.
However, if we ditch that too, and apply linebreaks everywhere (like you suggested earlier) it could be feasible.

We would essentially trade the linebreak-calculations in public data-revealing functions against a one-off linebreak addition on every change.